bin.src/validateDrp.py

```diff
@@ -51,7 +51,13 @@
                         help='YAML configuration file validation parameters and dataIds.')
     parser.add_argument('--verbose', '-v', default=False, action='store_true',
                         help='Display additional information about the analysis.')
-    
+    parser.add_argument('--plot', dest='makePlot', default=True,
+                        action='store_true',
+                        help='Make plots of performance.')
+    parser.add_argument('--noplot', dest='makePlot',
+                        action='store_false',
+                        help='Skip making plots of performance.')
+
```

Comment:
It seems that you only need `--noplot` here. Just add `default=True` to the `--noplot` argument and delete `--plot`.